lang/python/python3-flent/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-flent
```

Comment:
Not sure what the convention is on OpenWrt for Python packages, but on other distros the 'python-' prefix is for libraries, which Flent is not. So I think it would be better to just name the package 'flent'

Reply:
I am also not sure. Maybe I looked at the wrong packages, for example python3-speedtest-cli is also using the naming I used: https://github.com/openwrt/packages/blob/master/lang/python/python3-speedtest-cli/Makefile

Maybe @jefferyto can say something to that.

> So I think it would be better to just name the package 'flent'

Of course I can do that.